plugins/beers/beers.go

```diff
@@ -37,25 +37,22 @@ type untappdUser struct {
 	chanNick    string
 }
 
-// NewBeersPlugin creates a new BeersPlugin with the Plugin interface
+// New BeersPlugin creates a new BeersPlugin with the Plugin interface
```

Comment:
Thank god you fixed this

Reply:
No kidding. That linter was going crazy at me.